runner/database_test.go

```diff
@@ -177,7 +188,11 @@ func Test_getConnectionString(t *testing.T) {
 	for _, test := range tests {
 		vendor, connStr, err := ec.getConnectionString(test.conn)
 		assert.Equal(t, test.expVendor, vendor)
-		assert.Equal(t, test.expConnStr, connStr)
+		if test.expVendor == ODBCDatabase {
+			assert.Equal(t, nil, odbcAssert(test.expConnStr, connStr))
+		} else {
+			assert.Equal(t, test.expConnStr, connStr)
```

Comment:
If you need a completely new case then maybe just write a separate test rather than using this existing one?